docs/office-hours/20190709_meeting_notes.md

```diff
@@ -0,0 +1,97 @@
+
+Issue: https://github.com/osquery/foundation/issues/2
+
+# Platform support
+
+## Restoring previously supported platform
+
+The prerelease version only supports Ubuntu 18.04+, Mojave and Windows 10. In the upcoming stable version we will try to restore support for at least the following operating systems:
+
+ * CentOS 6, 7
+ * Ubuntu 16.04, 16.10
+ * Windows 7
+
+# Next stable release
+
+## Custom build toolchain
+
+The "custom toolchain" is a build toolchain to build libc++ in a reproducible manner (exact-hash reproducible), and statically link the same libc++ library to all distributions of the osquery executable.
+
+It is hard to ship binaries for Linux right now, as we basically have to compile them from scratch for each distribution we want to support. The custom build toolchain will solve this.
+
+## Security updates
+
+As the custom toolchain will make osquery depend on an old glibc version, in the future we should make sure that security updates on that library are applied.
+
+# Build system changes
+
+## Third-party dependencies
+
+Dependencies need to be moved to a place that is accessible by the Foundation. The best approach is to keep using S3, as bandwidth and data at rest have a reasonable price.
+
+The trouble with Git LFS is that it would require a lot of money (5 dollars for each 50 + 50 GiB of bandwidth and storage). Using git alone is not a really good alternative, as the repository would eventually become huge (and we likely also have a space limit).
+
+# Packaging
+
+## Signing keys
+
+We are not yet able to solve this problem; we could ask the original Facebook maintainers to sign the binaries manually. This is not always doable though. The easiest approach is for them to build from source.
+
+Keys are not in our possession yet, and things are still being negotiated with the Linux Foundation.
+
+Windows and macOS signing keys are likely not transferable, so new ones will be generated. In order to obtain them, osquery needs to become an entity that can then register with the relevant developer program (Apple, Microsoft).
+
+## Signing procedure
+
+It is helpful to separate the packaging procedure in multiple isolated steps.
+
+1. Build the project and generate a tarball
+2. Take the tarball and package the contents
+3. Take the package and sign it
+
+## Distro-specific repositories
+
+Packages are currently served via https from an S3 bucket on [pkg.osquery.io](https://pkg.osquery.io).
+
+A plan should be made to migrate them to a new bucket that is accessible by the foundation.
+
+## Chocolatey support
+
+The osquery package on Chocolatey is still owned by Facebook, and we are not able to update it.
+
+Supporting the ability to build MSI installers for osquery is a higher priority than supporting the Chocolatey package repository, but we would like to support both.
+
+# Funding
+
+## Open Collective
```

Comment:
FYI the LF has multiple ways of you raising money for your project in a neutral way, either through membership dues or via its https://communitybridge.org platform. We should sync up with what you want to do with raising money, but the individual donation approach doesn't work well, usually a dominant membership driven approach like CNCF or a mixed approached like letsencrypt.org that is hosted by the LF too works well.

Anyways, lets have a conversation about the project infra/costs needs and we can help.